schema.graphql

```diff
@@ -826,12 +899,14 @@ input UpdateWidgetInput {
   pluginId: PluginID!
   extensionId: PluginExtensionID!
   enabled: Boolean
+  layout: WidgetLayoutInput
 }
 
 input RemoveWidgetInput {
   sceneId: ID!
   pluginId: PluginID!
   extensionId: PluginExtensionID!
+  location: LocationInput!
```

Comment:
This is not needed because we can identify specific widget with only one widget ID

Reply:
The location is used to remove the widgetID from the align system. Without it, I need to create a function that will retrieve each WidgetArea and search its widgetIds until it finds the correct widgetId.
I thought that would be heavy so decided knowing the location was better.
How would you prefer this to be handled?

Reply:
That's not a problem because it's not a very heavy process. The information in the widget alignment system is always read and written together with the Scene, and all the information is put into memory at once. Their retrieval speed is usually not a concern at all (unless, for example, the process of scanning all the millions of records is repeated several times a second).

Reply:
Ok, I'll create such a function, remove this locationInput and update the RemoveWidget method.